Formula/lldb.rb

```diff
@@ -0,0 +1,83 @@
+class Lldb < Formula
+  desc "Next generation, high-performance debugger"
+  homepage "https://lldb.llvm.org"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-14.0.6/lldb-14.0.6.src.tar.xz"
+  sha256 "2ac41160b0bdc74bf72d4f9a3702f290f7c09998da6fb73281cc497f4ddbe130"
+  license "Apache-2.0" => { with: "LLVM-exception" }
+  head "https://github.com/llvm/llvm-project.git", branch: "main"
+
+  livecheck do
+    formula "llvm"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "sphinx-doc" => :build
+  depends_on "swig" => :build
+  depends_on "llvm"
+  depends_on "python@3.10"
+  depends_on "six"
+  depends_on "xz"
+
+  uses_from_macos "libedit"
+  uses_from_macos "libxml2"
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  on_macos do
+    keg_only "provided_by_macos" # TODO: change to :provided_by_macos after adding to Homebrew/brew
+  end
```

Comment:
Similar situation to `libxcrypt` (https://github.com/Homebrew/brew/pull/13157) where `:provided_by_macos` has deadlock for new formulae

Reply:
May not be necessary. This is probably more like `vim` than, say, `ncurses`, since macOS does not provide `liblldb`. It does provide `LLDB.framework`, but hides it away in a private location.